src/biokbase/narrative/jobs/job.py

```diff
@@ -11,106 +11,112 @@
 """
 __author__ = "Bill Riehl <wjriehl@lbl.gov>"
 
-EXCLUDED_JOB_STATE_FIELDS = ["authstrat", "job_input", "condor_job_ads"]
+COMPLETED_STATUS = "completed"
+TERMINAL_STATES = [COMPLETED_STATUS, "terminated", "error"]
+
+EXCLUDED_JOB_STATE_FIELDS = [
+    "authstrat",
+    "condor_job_ads",
+    "job_input",
+    "scheduler_type",
+    "scheduler_id",
+]
+
+EXTRA_JOB_STATE_FIELDS = ["batch_id", "cell_id", "run_id", "token_id"]
+
+JOB_DEFAULTS = {
+    "app_id": None,
+    "app_version": None,
+    "batch_id": None,
+    "batch_job": False,
+    "child_jobs": [],
+    "cell_id": None,
+    "params": None,
+    "meta": {},
+    "owner": None,
+    "run_id": None,
```

Comment:
Will need to add in `retry_count`, `retry_parent`, and `retry_ids` to this set of attributes once the ee2 API has stabilised.